Makefile

```diff
@@ -24,24 +24,24 @@ build_race:
 dist:
 	@BUILD_TAGS='$(BUILD_TAGS)' sh -c "'$(CURDIR)/scripts/dist.sh'"
 
-test:
-	@echo "--> Running linter"
-	@make metalinter_test
-	@echo "--> Running go test"
-	@go test $(PACKAGES)
+test: tools get_vendor_deps
+	echo "--> Running linter"
+	make metalinter_test
+	echo "--> Running go test"
+	go test $(PACKAGES)
 
 test_race:
-	@echo "--> Running go test --race"
-	@go test -v -race $(PACKAGES)
+	echo "--> Running go test --race"
+	go test -v -race $(PACKAGES)
 
-test_integrations:
-	@bash ./test/test.sh
+test_integrations: test
```

Comment:
also no

Reply:
seemed cleaner than adding a line in the `circle.yml`, but I get it, adds too much time